aptos-move/aptos-vm/src/transaction_validation.rs

```diff
@@ -216,6 +253,24 @@ pub(crate) fn run_script_prologue(
             .map_err(expect_no_verification_errors)
             .or_else(|err| convert_prologue_error(err, log_context))
     } else {
+        // Txn payload v2 format and orderless transactions are only supported with account abstraction.
```

Comment:
why it has to be with AA?

Reply:
I originally created `_v2` versions for all the prologue methods that will take replay proetector instead of a sequence number as input. But in our slack chat below, Igor mentioned to duplicate on unified_prologue methods as AA will be enabled before orderless transactions.

https://aptos-org.slack.com/archives/C07P622S5KN/p1738168487924729

Reply:
Done.